database/startup.js

```diff
@@ -0,0 +1,87 @@
+const faker = require('faker');
+const db = require('./index.js');
+const comments = require('./comments.js');
+
+const { connection } = db;
+
+const randomBoolean = () => Math.random() >= 0.5;
+
+const randomBound = (min, max) => Math.round(Math.random() * (max - min) + min);
+
+const randomComment = () => comments.comments[randomBound(0, comments.comments.length-1)];
+
+const createArtistData = () => {
+  for (let i = 0; i < 100; i += 1) {
+    const query = `INSERT INTO artists (userName, followers, followStatus, numTracks, dp) VALUES("${faker.internet.userName()}",${randomBound(100000, 1000000)},${randomBoolean()},${randomBound(5, 100)},"${faker.image.avatar()}")`;
```

Comment:
Very Long query line - for the sake of readability I might use backticks to give it newlines.